scripts/check_catalog_schema.py

```diff
@@ -39,6 +40,19 @@ def main(directory: Path) -> int:
             )
             code = 1
 
+        extra_fields = get_extra_fields(schema.field_names)
+
+        if len(extra_fields):
+            for extra_field in extra_fields:
```

Comment:
(Nit) Deux petites remarques:

* Pour les "conteneurs" (listes, sets, dicts...), `if len(x): ...` ou `if len(x) > 0: ...` est équivalent à `if x: ...`, et on préfère généralement cette dernière forme, plus "idiomatique".
* En réalité on n'a pas besoin du `if`, car `for x in <un set vide>` ça fonctionne :)

D'où :

```suggestion
        for extra_field in extra_fields:
```